spec/features/feed_spec.rb

```diff
@@ -0,0 +1,15 @@
+feature 'Feed' do
+  scenario 'A user can see a feed of all posted peeps' do
+    connection = PG.connect :dbname => "chitter_#{ENV['RACK_ENV']}"
+    user = User.new('Sandy', 'Pounoussamy', 'sandy974', 'pounoussamy.sandy@gmail.com', '1234')
+    peep = "Who wants to know what I had for lunch?!"
+    connection.exec("INSERT INTO peeps (user_id, posted_time, content) VALUES('#{user.id}', current_timestamp, '#{peep}')")
```

Comment:
In a feature test, you may want to only interact with your app through the user interface. So you could have your user create tow peeps, and then see them, for example.